src/task/MonitorTask.ts

```diff
@@ -228,45 +206,6 @@ export default class MonitorTask
         downlinkVideoStream.set(streamMetricReport[ssrc].streamId, streamMetricReport[ssrc]);
       }
     }
-
-    let fireCallback = false;
```

Comment:
These snippets no longer callback, and only log. They are not accurate when using the priority policy (before or after SSNA), and not worth fixing IMO, as since GA i have never seen these logs useful in my (or others) investigations, and they constantly become red herring when seen by builders. Let me know if anyone feels otherwise :)

Reply:
Yeah, they usually just cause confusions for devs.